src/mds/MDSDaemon.cc

```diff
@@ -462,25 +462,25 @@ void MDSDaemon::clean_up_admin_socket()
 
 int MDSDaemon::init()
 {
-  dout(10) << sizeof(MDSCacheObject) << "\tMDSCacheObject" << dendl;
-  dout(10) << sizeof(CInode) << "\tCInode" << dendl;
-  dout(10) << sizeof(elist<void*>::item) << "\t elist<>::item   *7=" << 7*sizeof(elist<void*>::item) << dendl;
+  dout(10) << sizeof(MDSCacheObject) << "\t MDSCacheObject" << dendl;
+  dout(10) << sizeof(CInode) << "\t CInode" << dendl;
+  dout(10) << sizeof(elist<void*>::item) << "\t elist<>::item * 7 = " << 7*sizeof(elist<void*>::item) << dendl;
```

Comment:
I don't know why Sage et al. added these scaling factors except perhaps they did some memory analysis and found the overhead was higher. However, I don't think it's desirable to ahve the scaling factor also printed to the log?

Reply:
Seems [this](https://github.com/ceph/ceph/commit/dc38d48948737be5b07814ddf198d4f2c14166bc) is the original commit and [this](https://github.com/ceph/ceph/commit/1278e4877633b52ed53baa219db518365f6eb425) one, but I'm not finding it's PR(s) (ancient) to check the reason for adding the scaling. Anyway, it's getting displayed several times in the logs whenever the MDSDaemon gets initialized and it should be the same. Yeah, the scale factor is obvious ;-)